doc/dev/ftrace.rst

```diff
@@ -0,0 +1,146 @@
+Tracing Kernel Functions with trace-cmd
```

Comment:
Make sure to link to this doc from `kclient.rst` in some new "Advanced debugging" section at the bottom of the guide.

Reply:
a67324ce968..29f186d3217 (which will probably be squashed by the time you see this... but maybe not)